Fixed the console  error in Imageview.dart

Saransh-cpp opened the pull request:
Closes #71 

Deleted the SafeArea and changed the flex for both the Expanded widgets which now are in a single column. This fixes the console error. Please let me know if you want any further changes in the PR. Thank you.

Saransh-cpp committed:
Fixed the console  error in Imageview.dart after an image is selected from gallery

anushbhatia reviewed (changes requested):
![image](https://user-images.githubusercontent.com/40017559/107143422-00207680-695b-11eb-830b-0ee40e0b113c.png)
Overflow error is occurring due to removing the safe area widget please solve this too

Saransh-cpp committed:
Fixed Renderflex overflow issues

Saransh-cpp commented:
Please don't merge right now, I am making some more changes.

Saransh-cpp committed:
Fixed the console error and renderflex overflow issue from Filterview.dart

Saransh-cpp committed:
Faded FadeTransition in ImageView and FilterView

Saransh-cpp commented:
Fixed everything now, I missed some Animated pushes and another console error and a following renderflex overflow from Filterview page. Opening other issue would have been repetitive so I fixed them in this PR. Let me know if you want any changes. Thank you.
